include/librealsense2/hpp/rs_processing.hpp

```diff
@@ -404,6 +404,37 @@ namespace rs2
         }
     };
 
+    class threshold_filter : public filter
```

Comment:
The name "threshold" is too generic, imo. this one is commonly addressed near/far distance clip